src/components/adjudications/AdjudicationsSection.jsx

```diff
@@ -42,27 +42,27 @@ class AdjudicationsSection extends React.Component {
     const { adjudications } = this.state;
     const collectionName = `events/${eventId}/performances/${performanceId}/adjudications`;
     const headings = ['Judge', 'Audio', 'Cummulative Score', 'Awards'];
+    const showAdjudications = Array.isArray(adjudications) && adjudications.length > 0;
 
     return (
       <React.Fragment>
         <SectionHeader title="adjudication" showNew={false} />
         <Table>
           <TableHeader headings={headings} />
           <TableBody>
-            {Array.isArray(adjudications) && adjudications.length
-              && adjudications.map((rowProps) => {
-                const keys = ['artisticMark', 'audio', 'choreoAward', 'cumulativeMark', 'notes', 'judgeName', 'sepcialAward', 'technicalMark'];
-                const currentValues = pick(rowProps, keys);
-                return (
-                  <AdjudicationTableRow
-                    collectionName={collectionName}
-                    currentValues={currentValues}
-                    key={rowProps.id}
-                    id={rowProps.id} />);
-              })}
+            {showAdjudications && adjudications.map((rowProps) => {
+              const keys = ['artisticMark', 'audio', 'choreoAward', 'cumulativeMark', 'notes', 'judgeName', 'specialAward', 'technicalMark'];
+              const currentValues = pick(rowProps, keys);
+              return (
+                <AdjudicationTableRow
+                  collectionName={collectionName}
+                  currentValues={currentValues}
+                  key={rowProps.id}
+                  id={rowProps.id} />);
+            })}
           </TableBody>
         </Table>
-        {!adjudications && (
+        {!showAdjudications && (
```

Comment:
Note: Let's do a check on adjudications.length

Reply:
@YangStephen not sure if we're on the same page, but `showAdjudications` comes from line 45:

`const showAdjudications = Array.isArray(adjudications) && adjudications.length > 0;` 

which checks the adjudications.length, is this what you mean?